index.js

```diff
@@ -6,6 +6,81 @@ var readline = require('readline');
 const Gamedig = require('gamedig');
 var token;
 
+// Define the commands.
+function BotCmd(cmd, desc) {
+    this.cmd  = cmd;
+    this.desc = desc;
+}
+const cmdSpacesSkip   = 5;
+const cmdHelpHeader   = "Hello world! This is VanillaBot! This is the The Ice Cream Team's first OFFICIAL SERVER (Get hyped BOOIIZZ) BOT! All interactions with this bot" +
+                        " begin with !vb and are followed by one or more arguments separated by spaces. For instance, to get information on the Minecraft server you" +
+                        " would type the following...\n\n!vb server mv\n\nBelow are the available arguments. Enjoy!\n";
+const cmdPrefix       = new BotCmd('!vb',          'Prefix needed for interacting with the bot.');
+const cmdServer       =   new BotCmd('server',     'Get information on one of Will\'s servers.');
+const cmdSvrMinecraft =     new BotCmd('mc',       'Specifies the minecraft server.');
+const cmdSvr7D2D      =     new BotCmd('7days',    'Specifies the 7 Days 2 Die.');
+const cmdPict         =   new BotCmd('pictionary', 'Returns the link to pictionary.');
+const cmdMal          =   new BotCmd('mal',        'Get information from MyAnimeList.');
+const cmdMalAnime     =     new BotCmd('anime',    'Find information on an anime. The next argument is the anime.');
+const cmdMalUser      =     new BotCmd('user',     'Find information on an user. The next argument is the user.');
+const cmdHelp         =   new BotCmd('help',       'Display information on how to utilize bot.');

```

Comment:
so after thinking about how the help looks, it's kinda outta whack/disorganized, i feel we can do this better to keep up consistency with the way we do the layout with our mal searching to make a bit more sense of the layout you have to someone who is new to using the bot (i do understand, and am ok with this spaced system, i just think it can be presented a bit better) its nothing we can change down the line, i might take a look at this after it's merged. 👍